sunpy/net/helioviewer.py

```diff
@@ -279,3 +357,5 @@ def _request(self, params):
     def _format_date(self, date):
         """Formats a date for Helioviewer API requests"""
         return parse_time(date).strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + "Z"
+
+
```

Comment:
I don't think we need two  blank lines here.

Reply:
Um is one blank line ok?

Reply:
Yeah.